frontend/src/components/Features.js

```diff
@@ -0,0 +1,17 @@
+import '../App.css';
+
+function Features() {
+  return (
+    <div id="features">
+        <div id = "featuresText">
+          What do we do?
+          <ul>
+              <li>Match animals with people who want to adopt</li>
+              <li>Help animals get adopted</li>
+          </ul>
+        </div>
+    </div>
+  );
+}
+
+export default Features;
```

Comment:
GitHub's mentioning that there's no newline at the end of the file, which is a tiny nitpick, so it's up to you on whether or not to change it.